quill-core/src/main/scala/io/getquill/quotation/Parsing.scala

```diff
@@ -353,11 +353,15 @@ trait Parsing extends SchemaConfigParsing {
   private def is[T](tree: Tree)(implicit t: TypeTag[T]) =
     tree.tpe <:< t.tpe
 
+  private def isTraversable(tree: Tree) = {
+    tree.tpe <:< typeOf[Traversable[_]]
```

Comment:
@getquill/maintainers This check is not strict. Is there any better alternative ?